experimental/dds/tree/package.json

```diff
@@ -0,0 +1,61 @@
+{
+	"name": "@fluidexperimental/tree",
+	"version": "0.17.1",
+	"description": "Distributed tree",
+	"homepage": "https://fluidframework.com",
+	"repository": "https://github.com/microsoft/FluidFramework",
+	"license": "MIT",
+	"author": "Microsoft",
+	"sideEffects": false,
+	"main": "dist/index.js",
+	"module": "lib/index.js",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"build": "concurrently npm:build:compile npm:lint",
+		"build:compile": "concurrently npm:tsc npm:build:esnext",
+		"build:docs": "api-extractor run --local",
+		"build:esnext": "tsc --project ./tsconfig.esnext.json",
+		"build:full": "npm run build",
+		"build:full:compile": "npm run build:compile",
+		"clean": "rimraf dist lib *.tsbuildinfo *.build.log",
+		"eslint": "eslint src",
+		"eslint:fix": "eslint src --fix",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prettier": "prettier --check \"**/*.{js,json,jsx,md,ts,tsx,yml,yaml}\"",
+		"prettier:fix": "prettier --write \"**/*.{js,json,jsx,md,ts,tsx,yml,yaml}\"",
+		"test": "npm run test:mocha",
+		"test:mocha": "nyc --report-dir='./coverage' mocha \"dist/**/*.tests.js\"",
+		"tsc": "tsc --pretty"
+	},
+	"dependencies": {
+		"@fluidframework/common-definitions": "^0.19.1",
+		"@fluidframework/common-utils": "^0.27.0-0",
+		"@fluidframework/container-definitions": "0.33.0",
+		"@fluidframework/core-interfaces": "0.33.0",
+		"@fluidframework/datastore-definitions": "0.33.0",
+		"@fluidframework/protocol-definitions": "^0.1018.0-0",
+		"@fluidframework/shared-object-base": "0.33.0",
+		"@fluidframework/telemetry-utils": "0.33.0",
+		"buffer": "^6.0.2",
+		"immutable": "^4.0.0-rc.12",
+		"sorted-btree": "^1.4.0",
+		"uuid": "^8.3.1"
+	},
+	"devDependencies": {
```

Comment:
You're missing deps on at least eslint and probably the other plug-in packages used by the shared config.

Reply:
Good catch -- fixed.  (We were picking them up artificially via Lerna hoisting.)